app/src/main/java/com/randomappsinc/studentpicker/grouping/GroupMakingSettingsViewHolder.java

```diff
@@ -70,4 +75,35 @@ private void applyNumberOfGroups() {
         }
         numGroups.clearFocus();
     }
+
+    @Override
+    public void beforeTextChanged(CharSequence s, int start, int count, int after) {}
```

Comment:
This is one of the very powerful parts of ButterKnife. You can use it to bind to view listening callbacks. This is how you listen to text changes without implementing 3 methods where you only use 1:
```
@OnTextChanged(value = R.id.search_input, callback = OnTextChanged.Callback.AFTER_TEXT_CHANGED)
public void afterTextChanged(Editable input) {
```
Just swap in the value with the EditText view you're binding to.